HTTP_API.md

```diff
@@ -1217,3 +1219,30 @@
     $ curl -X POST localhost:8181/api/v2/targets/localhost/snapshot
     {"downloadUrl":"http://192.168.0.109:8181/api/v1/targets/service:jmx:rmi:%2F%2F%2Fjndi%2Frmi:%2F%2Flocalhost:9091%2Fjmxrmi/recordings/snapshot-1","reportUrl":"http://192.168.0.109:8181/api/v1/targets/service:jmx:rmi:%2F%2F%2Fjndi%2Frmi:%2F%2Flocalhost:9091%2Fjmxrmi/reports/snapshot-1","id":1,"name":"snapshot-1","state":"STOPPED","startTime":1601998841300,"duration":0,"continuous":true,"toDisk":true,"maxSize":0,"maxAge":0}
     ```
+
+### Security
+
+* #### `CertificatePostHandler`
+
+    ###### synopsis
+    Uploads an SSL Certificate from the client, and saves it to the truststore directory.
+
+    ###### request
+    `POST /api/v2/certificates`
+
+    The certificate must be DER-encoded and can be either binary of base64. The supported extensions are .der, .cer, .pem. 
```

Comment:
typo: "can be either binary *of* base64"

Reply:
My bad